### PR TITLE
COTR-3 Coded player animations

### PR DESCRIPTION
**Pull request by Elijuszek**
Coded player IdleBack animation
Coded player IdleFront animation
Coded player IdleLeft animation
Coded player IdleRight animation
Coded player WalkBack animation
Coded player WalkFront animation
Coded player WalkLeft animation
Coded player WalkRight animation

### CODE REVIEW

File: Scenes/Actors/Troops/Infantry.cs [hunk 1]
```diff
@@ -4,12 +4,12 @@
 public partial class Infantry : Troop
 {
 	public MeleeAI AI { get; set; }
```
Comment (Karolinskis) [inline Scenes/Actors/Troops/Infantry.cs:6]: Add comments about this variable

File: Scenes/Actors/Troops/Infantry.cs [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
-
     public override void _Ready()
     {
         base._Ready();
         AI = GetNode<MeleeAI>("MeleeAI");
     }
 
     public virtual void Attack() { }
+    public virtual void RotateAttack() { }
```
Comment (Karolinskis) [inline Scenes/Actors/Troops/Infantry.cs:14]: Add summaries of these methods

**Review by Karolinskis — CHANGES_REQUESTED** [review]

File: Scenes/Actors/Troops/Infantry.cs (hunk 1 above)
Comment (Elijuszek) [reply Scenes/Actors/Troops/Infantry.cs:6]: Done

File: Scenes/Actors/Troops/Infantry.cs (hunk 2 above)
Comment (Elijuszek) [reply Scenes/Actors/Troops/Infantry.cs]: Done

**Review by Karolinskis — APPROVED** [review]

**Pull request merged**